### DISCUSSION TITLE
Welcome to samat Discussions!

### DISCUSSION THREAD

**Discussion by Divelix** [Divelix/samat#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by paujar** [comment]
I am wondering that ;

Note: Magic Wand is optional tool and uses SAM masks that can be generated by python script in scripts directory of this repo.

So to use SAM this magic wand requires SAM, yes, and for that I need to run preprocess_dataset.py with pretrained SAM weights, where can I obtain those? Couldn't find (at least yet)

**Reply by Divelix** [reply]
Hi, you can find them in [SAM readme](https://github.com/facebookresearch/segment-anything?tab=readme-ov-file#model-checkpoints).
